nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1600,23 +1506,9 @@ define(
             }
 
             function doExitCancelled() {
-                if (widgets.runClock) {
-                    widgets.runClock.stop();
-                }
                 ui.setContent('run-control-panel.execMessage', '');
             }
 
-            function doRemove() {
-                const confirmationMessage = div([p('Continue to remove this app cell?')]);
-                ui.showConfirmDialog({ title: 'Remove Cell?', body: confirmationMessage }).then(
-                    (confirmed) => {
-                        if (!confirmed) {
-                            return;
-                        }
-                    }
-                );
-            }
```

Comment:
could not find anywhere that called the code that calls this function, and it doesn't even do anything if you do click 'OK'!